libs/cms/src/lib/models/organizationParentSubpage.model.ts

```diff
@@ -23,6 +23,9 @@ export class OrganizationParentSubpage {
   @Field()
   title!: string
 
+  @Field(() => String, { nullable: true })
+  shortTitle?: string
```

Comment:
I wonder if we should also add this to the GraphQL query but I guess there isn't anything using the field so perhaps not